### ISSUE TITLE
Traefik Dashboard on private network

### ISSUE THREAD

**Issue by ghost** [spantaleev/gitea-docker-ansible-deploy#4]
I would like to run Traefik Dashboard on my local network only, but I'm doing something wrong maybe.

my vars file:
```yml
gitea_playbook_docker_installation_enabled: false

# Traefik Dashboard Insecure Mode
devture_traefik_configuration_extension_yaml: | 
  api:
    dashboard: true
    insecure: true
```
After execute `ansible-playbook -i inventory/hosts setup.yml --tags=setup-all,start` nothing changes and dashboard is not available from `http://<Traefik IP>:8080/dashboard/`.

[Insecure Mode](https://doc.traefik.io/traefik/operations/dashboard/#insecure-mode) from Traefik Dashboard Documentation.

**Comment by spantaleev** [comment]
Maybe that 8080 port needs to be exposed outside of the container network?

You can use `devture_traefik_container_extra_arguments: ['-p 8080:8080']` to expose this 8080 container port on **all** network interfaces. On the container side, the port also needs to be published on 0.0.0.0, but I suppose this may already be the case by default.

**Comment by ghost** [comment]
Unfortunately, even with the added line `devture_traefik_container_extra_arguments: ['-p 8080:8080']` I don't see port 8080 exposed...
